harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/AbstractHarvestError.java

```diff
@@ -0,0 +1,24 @@
+package org.fao.geonet.kernel.harvest.harvester;
+
+import org.fao.geonet.monitor.harvest.AbstractHarvesterErrorCounter;
+
+import com.yammer.metrics.core.Counter;
+
+import jeeves.monitor.MonitorManager;
+import jeeves.server.context.ServiceContext;
+
+public class AbstractHarvestError {
+	
+	public AbstractHarvestError(ServiceContext context) {
+		super();
+		// we don't catch anything because this is not critical
+		// if the following code is not executed, we just want to
+		// avoid to raise an exception.
+		try {
+			MonitorManager mm = context.getBean(MonitorManager.class);
```

Comment:
We're trying to reduce the dependency with Jeeves, so could we replace that ServiceContext by ApplicationContextHolder.get().getBean instead ? or even use ```@autowired``` on MonitorManager (AFA as multinode monitor managers are probably not used - ie we have one monitor bean for all ?)